docs/sync-storage-handbook/src/places.md

```diff
@@ -0,0 +1,35 @@
+# Places
+
+Places is a library for storing and syncing bookmarks and history. It exposes high-level APIs for common use cases, like recording history visits, autocompleting visited URLs, clearing and expiring history, and managing bookmarks. It also provides engines for syncing visited pages and bookmarks through Firefox Sync.
```

Comment:
I wonder if it's obvious to the new reader why we cram history and bookmarks into a single store. Is there anything we can say here to motivate that?  ("Because that's how Desktop places does it" is fine, but why does Desktop places do that?)